Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx

```diff
@@ -161,6 +161,21 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
     }
   }
 
+  private sendingNumbers = (message: any) => {
+    let currentNum = message.succeeded + message.failed + message.throttled;
+    if (message.isCompleted) {
+      return (<Text />);
+    } else {
+      return (
+        <Text
+          truncated
+          content={`Sending...  ${currentNum} / ${message.totalMessageCount}`}
```

Comment:
Design said they actually want this to look like "Sending... 50 of 1000"
Just changing the '/' to the word "of".

Reply:
done